### PR TITLE
Adding logging for debugging slave lag

### PR DESCRIPTION
**Pull request by camilo**
@jasonhl @sroysen

### CODE REVIEW

**Comment by sroysen** [comment]
:+1:

**Comment by camilo** [comment]
@arthurnn yo ok with this, just adding logging for some fun debugging times

File: lib/lhm/throttler/slave_lag.rb [hunk 1]
```diff
@@ -53,7 +55,9 @@ def get_slaves
       end
 
       def max_current_slave_lag
-        slave_hosts.map { |slave| slave_lag(slave) }.flatten.push(0).max
+        max = slave_hosts.map { |slave| slave_lag(slave) }.flatten.push(0).max
+        LHM.logger.info "Max current slave lag: #{slaves.join(',')}"
```
Comment (arthurnn) [inline lib/lhm/throttler/slave_lag.rb]: s/LHM/Lhm

**Comment by camilo** [comment]
lol fixed

File: lib/lhm/throttler/slave_lag.rb [hunk 2]
```diff
@@ -53,7 +55,9 @@ def get_slaves
       end
 
       def max_current_slave_lag
-        slave_hosts.map { |slave| slave_lag(slave) }.flatten.push(0).max
+        max = slave_hosts.map { |slave| slave_lag(slave) }.flatten.push(0).max
+        Lhm.logger.info "Max current slave lag: #{slaves.join(',')}"
```
Comment (jasonhl) [inline lib/lhm/throttler/slave_lag.rb]: There is no `slaves` variable at this point?

Comment (arthurnn) [inline lib/lhm/throttler/slave_lag.rb]: right.. CI is failling because of this line.

**Comment by camilo** [comment]
> Lhm::changes::parallel#test_0002_should perserve deletes during migration:
ActiveRecord::StatementInvalid: Mysql2::Error: This connection is in use by: #<Thread:0x000000029ba580 sleep>: SELECT DATABASE() as db

^ this seems unrelated

**Comment by arthurnn** [comment]
sweet. squash and I will merge it.
thanks

**Comment by camilo** [comment]
HULK SQUASH

image

**Comment by camilo** [comment]
Smething is fucky with the tests, I think the connections might be shared, but seems unrelated, ok to merge now @arthurnn ?

**Pull request merged**